### PR TITLE
feat : added Toast notification

### PR DESCRIPTION
**Pull request by Tanjum786**
added toast for

- add to the cart 
- add to the wishlist
- remove form cart or wishlist 

https://user-images.githubusercontent.com/84380051/162716157-83b3c729-a7c0-4933-b810-c179b671532b.mp4

### CODE REVIEW

File: src/components/Navbar-comp/Navbar.jsx [hunk 1]
```diff
@@ -1,7 +1,10 @@
 import React from "react";
 import { Link } from "react-router-dom";
+import { useProducts } from "../../hooks/context/cart-wishlistcontext";
 
 export const Navbar = ({ auth,linkto }) => {
+  const {productState}=useProducts();
+  const {mycart,mywishlist}=productState;
```
Comment (vaibhav-mahalle) [inline src/components/Navbar-comp/Navbar.jsx]: you can use camelcasing here for better naming e.g. myCart

File: src/hooks/context/Reducer/CartRedcerFun.jsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
+export const CartRedcerFun = (cartstate, cartaction) => {
+  const { mycart, mywishlist } = cartstate;
```
Comment (vaibhav-mahalle) [inline src/hooks/context/Reducer/CartRedcerFun.jsx]: same as above.

File: src/pages/Cart/component/Cardcart.jsx [hunk 3]
```diff
@@ -1,44 +1,92 @@
...
-  cartimg,
-  cutprice,
-  discount,
-  Quantity,
-}) => {
+import { useProducts } from "../../../hooks/context/cart-wishlistcontext";
+
+const Cardcart = () => {
+  const { productState, productDispatch } = useProducts();
+  const { mycart } = productState;
```
Comment (vaibhav-mahalle) [inline src/pages/Cart/component/Cardcart.jsx]: same as mentioned above.

File: src/pages/Wishlist/component/Wishlistcard.jsx [hunk 4]
```diff
@@ -1,27 +1,65 @@
...
+                        rating: rating,
+                        Quantity: Quantity,
+                      },
+                    })
+                  }
+                >
+                  <i className="fas fa-heart"></i>
+                </button>
+              </div>
+              <img src={img} alt="img" />
```
Comment (vaibhav-mahalle) [inline src/pages/Wishlist/component/Wishlistcard.jsx]: you can mention which type of image it is e.g product_img

**Review by vaibhav-mahalle — APPROVED** [review]
added some suggestions please look into them.

**Review by raman1999 — APPROVED** [review]
loos good, can merge

**Pull request merged**